lib/bandiera/caching_feature_service.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'lru_redux'
+
+module Bandiera
+  class CachingFeatureService < SimpleDelegator
+    CACHE_SIZE = 100
+    CACHE_TTL  = 10
+
+    attr_reader :cache
+
+    def initialize(delegate)
+      @cache = LruRedux::TTL::ThreadSafeCache.new(CACHE_SIZE, CACHE_TTL)
```

Comment:
Just a question - when rolling a load-balanced Bandiera setup (currently how we use it at @ukparliament)

Each node will create their own in memory cache right?

Reply:
Yep, that's correct.

Reply:
Looks great, thanks - very happy to replace custom code with something battle tested 😄